BitmovinPlayerSSAI/BitmovinPlayerSSAI/ssai/BitmovinPlayerSsaiAdScheduler.swift

```diff
@@ -0,0 +1,146 @@
+import BitmovinCollector
+import BitmovinPlayer
+import CoreCollector
+
+class BitmovinPlayerSsaiAdScheduler: NSObject {
+
+    static let beforeAdTimeSpan = 200
+
+    private let player: Player
+    private let collector: PlayerAnalyticsApi
+    private var adsScheduled: [SsaiAdInfo] = []
+    private var activeAd: SsaiAdInfo?
+    private var nextAd: SsaiAdInfo?
+
+    init(player: any Player, collector: any PlayerAnalyticsApi) {
+        self.player = player
+        self.collector = collector
+    }
+
+    /**
+     Schedules the ads.
+     Listens to the player events to know when ads should be tracked
+     */
```

Comment:
This is the old C/ObjC way of doing comments. Swift uses `///` **all occurrences**

```suggestion
    /// Schedules the ads.
    /// Listens to the player events to know when ads should be tracked
```

Reply:
fixed: https://github.com/bitmovin/bitmovin-analytics-collector-ios-samples/pull/8/commits/ea60178c7733044ae3ebfaa292154f7721f236b4